pkg/cmd/server/origin/auth.go

```diff
@@ -320,23 +320,26 @@ func (c *AuthConfig) getAuthenticationHandler(mux cmdutil.Mux, errorHandler hand
 		identityMapper := identitymapper.NewAlwaysCreateUserIdentityToUserMapper(c.IdentityRegistry, c.UserRegistry)
 
 		if configapi.IsPasswordAuthenticator(identityProvider) {
-			passwordAuth, err := c.getPasswordAuthenticator(identityProvider)
```

Comment:
go ahead and revert this part, we can look at adjusting how the origin login page looks, but the capability needs to remain in place